add header mdx (#310)
allenai/varnish-legacy: demo/src/pages/Header/demo/withMenu.mdx

---
order: 1
title: With Menu
---

> Header inside a layout with a logo, menu, and content.

Reviewer: ```suggestion
This shows how to render a header with a logo and navigation.
```
